src/App.js

```diff
@@ -1,23 +1,46 @@
-import logo from './logo.svg';
 import './App.css';
+import React, { useState } from 'react';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFoodForm from './components/AddFoodForm';
+import Search from './components/Search';
+
 
 function App() {
+  const [dishes, setDishes] = useState(foods)
```

Comment:
For the Search function to work properly you need to add a new `useState`, so under the `useState` that you already have you can add this line:
` const [showFoods, setShowFoods] = useState(foods);`